RelationalAI/RelationalAI.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>0.9.9-alpha</Version>
+    <Version>0.9.10-alpha</Version>
```

Comment:
whats up with this and the Protobuf changes? These are merged right, perhaps just a github artifact?  Approved overall for the PR just please verify we dont have a merge issue here